cmd/mrpeek.cpp

```diff
@@ -580,23 +588,26 @@ std::string display (ImageType& image, Sixel::ColourMaps& colourmaps)
 
     if (!cmap.scaling_set())
       autoscale (image, cmap);
-
-    out += ClearLine;
-    if (arrow_mode == ARROW_COLOUR)
-      out += TextForegroundYellow;
-    out += str(cmap.max(),4) + TextReset + move_down(1) + position_cursor_at_col (2);
+    if (show_text) {
+      out += ClearLine;
+      if (arrow_mode == ARROW_COLOUR)
+        out += TextForegroundYellow;
+      out += str(cmap.max(),4) + TextReset + move_down(1) + position_cursor_at_col (2);
+    }
 
     out += display_image (image, cmap, 2*COLOURBAR_WIDTH) + CarriageReturn + ClearLine;
 
-    if (arrow_mode == ARROW_COLOUR)
-      out += TextForegroundYellow;
-    out += str(cmap.min(), 4) + TextReset + move_down(1) + CarriageReturn;
+    if (show_text) {
+      if (arrow_mode == ARROW_COLOUR)
+        out += TextForegroundYellow;
+      out += str(cmap.min(), 4) + TextReset + move_down(1) + CarriageReturn;
+    }
   }
 
 
-  out += show_focus(image);
+  if (show_text) out += show_focus(image);
 
-  if (orthoview) {
+  if (interactive and orthoview and show_text) {
```

Comment:
Been writing a lot of python lately...?

Reply:
```c++
%:include <iostream>
 
struct X
<%
    compl X() <%%> // destructor
    X() <%%>
    X(const X bitand) = delete; // copy constructor
 
    bool operator not_eq(const X bitand other)
    <%
       return this not_eq bitand other;
    %>
%>;
 
int main(int argc, char* argv<::>) 
<%
    // lambda with reference-capture:
    auto say = <:bitand:>(const char* name)
    <%
        std::cout << "I like " << name << '\n';
    %>;
 
    if (argc > 1 and argv<:1:> not_eq nullptr) <%
        say(argv<:1:>);
    %>
    else <%
        say("Python");
    %>
%>
```

Reply:
Wow, I had no idea there was such a thing as [the alternative operators](https://en.cppreference.com/w/cpp/language/operator_alternative)... Guess you learn something new every day!